src/app/views/panels/widgets/controls/styles.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import styled from "styled-components";
+import { ITextFieldStyles, ITooltipHostStyles } from "@fluentui/react";
+
+export const ImageMappingDragStateWrapper = styled.div`
```

Comment:
Consider removing `ImageMappingDragStateWrapper` unless used elsewhere in the code.

Reply:
`ImageMappingDragStateWrapper` is used in several places in the code.